tools/windows-node-installer/pkg/cloudprovider/aws/ec2_test.go

```diff
@@ -89,7 +130,7 @@ func TestGetRulesForSgUpdate(t *testing.T) {
 			expected: []*ec2.IpPermission{
 				getAllPortRule(vpcCidr),
 				getOtherPortRules(WINRM_PORT, myIP),
-				getOtherPortRules(sshPort, myIP),
+				getOtherPortRules(types.SshPort, myIP),
```

Comment:
I'll remove this when the #187 merges

Reply:
please add /hold to the PR. I'm afraid this might get merged before #187